griptape/loaders/pdf_loader.py

```diff
@@ -16,10 +16,10 @@ class PdfLoader(TextLoader):
         kw_only=True,
     )
 
-    def load(self, stream: str | IO | Path, password: Optional[str] = None) -> list[TextArtifact]:
+    def load(self, stream: str | IO | Path, password: Optional[str] = None) -> list[TextArtifact]:  # type: ignore
         return self._load_pdf(stream, password)
 
-    def load_collection(
+    def load_collection(  # type: ignore
```

Comment:
Let's resolve these type errors while we're already in here. What was the pyright output?

Reply:
Or if you'd like to help resolve https://github.com/griptape-ai/griptape/issues/585, we can merge that in first.

Reply:
Hmm ok this probably requires a bit of thinking on how we want to structure this. Let's still remove the `type: ignore` so we don't mask the errors. You can skip the git precommit hooks with the `--no-verify` commit flag.